### PR TITLE
Automatically build with Travis CI

### PR DESCRIPTION
**Pull request by bin3377**
This change will trigger following actions on Travis CI:
- Unit test
- Build Gem
- Build docker image with built-out Gem (not source file)
- Push to docker hub with new tag (with GitHub Tag build only, version number from GitHub tag, format as `v1.0.1` and "latest")
- Push to Ruby gems with new tag (with GitHub Tag build only, version number from GitHub tag, format as `1.0.1`)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -61,10 +61,10 @@ ENV K8S_METADATA_FILTER_BEARER_CACHE_TTL "3600"
 ENV VERIFY_SSL "true"
 
 COPY --from=builder /var/lib/gems /var/lib/gems
-
 COPY ./conf.d/ /fluentd/conf.d/
 COPY ./etc/* /fluentd/etc/
-COPY ./lib/fluent/plugin/* /fluentd/plugins/
 COPY ./entrypoint.sh /fluentd/
+COPY ./fluent-plugin-kubernetes_sumologic*.gem ./
+RUN gem install fluent-plugin-kubernetes_sumologic
```
Comment (frankreno) [inline Dockerfile]: Should this be here or in the base image? The reason for the layer builds was to remove the build tools not needed in the final image, see RUN on line 6

Comment (bin3377) [reply Dockerfile]: I'm not very sure about this. [The example](https://github.com/fluent/fluentd-docker-image#31-for-current-images) from fluentd doesn't use any builder image. I can give it a try

Comment (bin3377) [reply Dockerfile]: Looks like it works. Move it to builder

**Comment by bin3377** [comment]
FYI - @bendrucker

**Comment by bin3377** [comment]
@frankreno if there is no more comments, I will merge it later today and create a new release for testing

**Pull request merged**